README.md

```diff
@@ -236,7 +236,7 @@ React will automatically check the propTypes you set on the component, but if
 you are using PropTypes without React then you may want to manually call
 `PropTypes.checkPropTypes`, like so:
 
-```
+```js
```

Comment:
For some reason, 'jsx' syntax highlighting works much better than 'js' syntax highlighting for objects. So I tend to opt for that, as well as incase one need to have JSX in the code snippet in future as well.